dpl-cloud_files.gemspec

```diff
@@ -1,3 +1,7 @@
 require './gemspec_helper'
 
-gemspec_for 'cloud_files', [['net-ssh'], ['mime-types'], ['nokogiri'], ['fog-rackspace']]
+if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.3.0")
```

Comment:
This is not really effective.

`*.gemspec` is read only once when the gem is built. This means that the resultant gem would have different requirements based on the Ruby version that built it.

This can be a source of confusion if different people build and publish them.

So please don't do this.

Reply:
I see. Was just following what I saw in the `chef_supermarket` spec, but perhaps that has the same problem.

Reply:
Indeed, I was naïve back then. chef_supermarket shouldn't do that, either. 😬